frontend/src/App.js

```diff
@@ -1,30 +1,45 @@
-import React from "react";
+import React, { Component } from "react";
 import { BrowserRouter as Router, Route, Link } from "react-router-dom";
 import "./App.css";
 import Status from "./Status";
 import Home from "./Home";
 import Experiment from "./Experiment";
 import Emib from "./components/eMIB/Emib";
+import Translation from "./components/commons/Translation";
+import LOCALIZE from "./text_resources";
 
-const App = () => {
-  return (
-    <div className="app">
-      <Router>
-        <div>
-          <div>
-            <Link to="/">Home</Link>
-            <Link to="/experiment">Prototype</Link>
-            <Link to="/status">Status</Link>
-          </div>
-          <hr />
-          <Route exact path="/" component={Home} />
-          <Route path="/experiment" component={Experiment} />
-          <Route path="/status" component={Status} />
-          <Route path="/emib-sample" component={Emib} />
+class App extends Component {
+  state = {
+    language: "en"
+  };
+
+  updateLanguage = () => {
+    this.setState({ language: "fr" });
```

Comment:
this should only set it to "fr" if the current state is "en", and only set it to "en" if the current state is "fr".

Reply:
also instead of using strings, you should import `LANGUAGES` from `Translation.jsx` and use the variables.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/63/commits/bd78c694b0a6ab88f4f9727a31b951e1f2529977